templates/src/objective-c/BoringSSL-GRPC.podspec.template

```diff
@@ -209,25 +219,26 @@
         }
       EOF
 
-      # The symbol prefixing mechanism is performed by redefining BoringSSL symbols with "#define
-      # SOME_BORINGSSL_SYMBOL GRPC_SHADOW_SOME_BORINGSSL_SYMBOL". Unfortunately, some symbols are
-      # already redefined as macros in BoringSSL headers in the form "#define SOME_BORINGSSL_SYMBOL
-      # SOME_BORINGSSL_SYMBOL" Such type of redefinition will cause "SOME_BORINGSSL_SYMBOL redefined"
-      # error when using together with our prefix header. So the workaround in the below lines removes
-      # all such type of #define directives.
-      sed -i'.back' '/^#define \\([A-Za-z0-9_]*\\) \\1/d' src/include/openssl/*.h
-      # Remove lines of the format below for the same reason above
-      #     #define SOME_BORINGSSL_SYMBOL ${"\\"}
-      #         SOME_BORINGSSL_SYMBOL
-      sed -i'.back' '/^#define.*\\\\$/{N;/^#define \\([A-Za-z0-9_]*\\) *\\\\\\n *\\1/d;}' src/include/openssl/*.h
+      # To avoid symbol conflict with OpenSSL, gRPC needs to rename all the BoringSSL symbols with a 
+      # prefix. This is done with BoringSSL's BORINGSSL_PREFIX mechanism
+      # (https://github.com/google/boringssl/blob/75148d7abf12bdd1797fec3c5da9a21963703516/BUILDING.md#building-with-prefixed-symbols).
+      # The required prefix header file boringssl_prefix_symbols.h is not part of BoringSSL repo at
+      # this moment. It has to be generated by BoringSSL's users and be injected to BoringSSL build.
+      # gRPC generates this file in script /tools/distrib/upgrade_boringssl_objc.sh. This script
+      # outputs a gzip+base64 encoded version of boringssl_prefix_symbols.h because of Cocoapods'
+      # limit on the 'prepare_command' field length. The encoded header is generated from
+      # /src/boringssl/boringssl_prefix_symbols.h. Here we decode the content and inject the header to
+      # the correct location in BoringSSL.
+      base64 -D <<EOF | gunzip > src/include/openssl/boringssl_prefix_symbols.h
+        ${prefix_gz_b64}
```

Comment:
nit: I looked at the generated file and it looks like it generates one extremely long line with base64.
It would be nicer if you wrapped the base64 string at 80 characters, which should still be parseable by the base64 command, but the generated file would look that ugly?